src/app/components/home.component.ts

```diff
@@ -135,6 +135,7 @@ export class HomeComponent implements OnInit, AfterViewInit {
   versionNumber = globals.version;
 
   public finalArray: ImageElement[] = [];
+  public selectedCount: number;
```

Comment:
I like the idea of `selectedCount` -- are you thinking of using it somewhere in the _template_ (to show to the user?)